server/services/auth_service.py

```
@@ -26,14 +27,16 @@ async def signup(self, signup_data: SignupRequest) -> SignupResponse:
-        # Call domain for business logic validation
-        self.auth_domain.validate_signup_data(signup_data)
-        
-        # Repository interface - create user
-        result = await self.auth_repository.create_user(signup_data)
+        try:
+            self.auth_domain.validate_signup_data(signup_data)
+            result = await self.auth_repository.create_user(signup_data)
+        except InputValidationError:
+            raise
+        except Exception as e:
+            raise DatabaseError("Failed to create user") from e
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)